src/Collection/AuditedCollection.php

```diff
@@ -20,6 +20,12 @@
 use SimpleThings\EntityAudit\AuditReader;
 use SimpleThings\EntityAudit\Exception\AuditedCollectionException;
 
+/**
+ * @phpstan-template TKey
+ * @psalm-template TKey of array-key
+ * @psalm-template T
```

Comment:
Why do you need different template for phpstan and psalm ?

Reply:
It was copied from `Collection`.